deployment/roles/monitor/tasks/cron.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Parse cron schedule
+  set_fact:
+    minute: "{{ MONITOR_CRON_SCHEDULE.split(' ')[0] }}"
```

Comment:
How does this work when `MONITOR_CRON_SCHEDULE` is not set?

Reply:
If `MONITOR_CRON_SCHEDULE` is not set or invalid, task fails.
The playbook continues because `ignore_errors: True`.
The `Add cron entry` is then skipped because `when: not set_fact_result.failed`

Reply:
I think we should always install cron for monitor (otherwise it just checks ones), and so add a default for `MONITOR_CRON_SCHEDULE`

Reply:
I agree. I made the changes in the playbook and the docs, and also renamed `MONITOR_CRON_SCHEDULE` to `monitor_cron_schedule` to match style.